lib/instrumentation.ex

```diff
@@ -72,6 +73,20 @@ defmodule OpentelemetryAbsinthe.Instrumentation do
   end
 
   def handle_operation_stop(_event_name, _measurements, data, config) do
+    fields =
+      case data do
+        %{blueprint: %{operations: [_ | _] = operations}} ->
+          operations
+          |> Enum.reduce([], fn selections, list ->
+            selections.selections
+            |> Enum.reduce(list, fn selection, list -> [selection.name | list] end)
+          end)
```

Comment:
I find this double `reduce` a bit confusing, it seems it could be achieved with a `map` and a `flat_map` but I'm not sure

Reply:
Done.. agreed, this reads cleaner now.